bin/check.sh

```diff
@@ -19,7 +19,7 @@ gometalinter --concurrency=${NUM_CPU} --enable-gc --deadline=300s --disable-all\
   --enable=gotype\
   --enable=ineffassign\
   --enable=interfacer\
-  --enable=lll --line-length=120\
+  --enable=lll --line-length=160\
```

Comment:
where does this stop ?

Reply:
It stops when we have a consistent Istio project - there is no reason for mixer and istio to have a different code style from pilot. 

I would note that google go style, and normal go style doesn't have limits on line length - not clear why Istio needs to be different, or how such decision is made - but 160 wins.

A bigger problem is the ineffassign - which seems to prevent 
port := 0 - for no good reason. I think we need some process around such decisions.

Reply:
Well, I'm a vim user and I like 120 characters. 160 is too long. Don't bundle these subtle checks into PRs, it's easy to miss and will surprise us later.

Reply:
+1

Reply:
re ineffassign, it's for a good reason it is not doing anything... it should be
```
	var port int
```
or better deleted and := in each block

what's the problem with that ?

Reply:
Reverted to 120 until we can have a discussion on style guides for Istio (instead of each repo doing its own random stuff).

Reply:
Re. ineffassign: original code (port:=0) fails the linter, it has to be changed to "var port int". 
The problem is that port:=0 is a perfectly common pattern, and shouldn't be rejected.

Reply:
bikeshedding here but having a variable defined at the minimum scope required is also a good thing (in this case  port wasn't really needed at that scope - eg proper fix should be https://github.com/istio/pilot/pull/1121/files)

one benefits of linter is there is no discussion/argument: just do what the machine tells you :-)

Reply:
Still see 160